libraries/AP_OSD/AP_OSD.h

```diff
@@ -34,7 +34,7 @@
 #endif
 
 #ifndef OSD_PARAM_ENABLED
-#define OSD_PARAM_ENABLED !HAL_MINIMIZE_FEATURES
+#define OSD_PARAM_ENABLED OSD_ENABLED && !HAL_MINIMIZE_FEATURES
```

Comment:
This is not correct. The whole purpose fo the PR was to enable OSD_PARAM_ENABLED without OSD_ENABLED needing to be enabled

Reply:
I see, failing on omnibusf4pro when you set HAL_MINIMIZE_FEATURES to 1 in the hwdef

Reply:
Thanks I will take a look